assets/js/blocks/cart-checkout/checkout-i2/inner-blocks/checkout-actions-block/index.tsx

```diff
@@ -21,6 +21,12 @@ registerFeaturePluginBlockType( 'woocommerce/checkout-actions-block', {
 		align: false,
 		html: false,
 		multiple: false,
+		reusable: false,
```

Comment:
Removes this button
![image](https://user-images.githubusercontent.com/6165348/124760936-27d73300-df29-11eb-8491-74cea2631f56.png)